corehq/apps/app_manager/suite_xml/sections/remote_requests.py

```diff
@@ -131,8 +131,13 @@ def _build_remote_request_datums(self):
             long_detail_id = 'search_long'
 
         nodeset = CaseTypeXpath(self.module.case_type).case(instance_name=RESULTS_INSTANCE)
-        if self.module.search_config.search_filter and toggles.USH_CASE_CLAIM_UPDATES.enabled(self.app.domain):
-            nodeset = f"{nodeset}[{interpolate_xpath(self.module.search_config.search_filter)}]"
+        if toggles.USH_CASE_CLAIM_UPDATES.enabled(self.app.domain):
+            additional_types = list(set(self.module.additional_case_types) - {self.module.case_type})
+            if additional_types:
+                nodeset = CaseTypeXpath(self.module.case_type).cases(
+                    additional_types, instance_name=RESULTS_INSTANCE)
+            if self.module.search_config.search_filter:
+                nodeset = f"{nodeset}[{interpolate_xpath(self.module.search_config.search_filter)}]"
```

Comment:
Should this use a `+=`? It looks like if there are additional types and also a search filter, the final nodeset won't incorporate both.

Reply:
It would, I think since `f"{nodeset}` includes the nodeset prior, no?

Reply:
aha, I misread